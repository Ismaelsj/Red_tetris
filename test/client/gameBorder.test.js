import '../jsDomSetup'
import React from 'react'
import { shallow } from 'enzyme';
import "babel-polyfill"

import GameBorder from '../../src/client/containers/gameBoard/gameBorder'

describe('Game border =>', () => {

    test('game border :', () => {
        const component = shallow(
            <GameBorder />
        );

        expect(component).toMatchSnapshot()
    });
}) 