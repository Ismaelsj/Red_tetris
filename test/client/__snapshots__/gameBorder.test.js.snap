// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game border => game border : 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
